tools/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["plugin:@fluentui/eslint-plugin/node"],
+  "rules": {
+    "import/no-extraneous-dependencies": "off"
```

Comment:
I guess anything in `tools` will not declare dependencies in `package.json` and just consume from the root right ?

Reply:
yes, single version policy from the start.

- also because this will never be published
  - there is no need to provide deps
  - if we might change our minds, nx generates package.json with deps automatically (based on AST/dependency graph)